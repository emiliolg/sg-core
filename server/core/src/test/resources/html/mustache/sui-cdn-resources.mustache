<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views">
<head>
    <title>Resources examples</title>
    <link rel="icon" href="{{#favicon}}{{/favicon}}"/>
    <link href="{{#css}}theme.css{{/css}}">
    <link rel="stylesheet" href="{{#sha}}/css/alpha.css,
            /css/beta.css
                {{/sha}}">
    <script src="{{#js}}lib.js{{/js}}"></script>
</head>
<body>
<img src="{{#img}}image.jpeg{{/img}}">
<img src="{{#img}}/some/image.jpeg{{/img}}">
<img src="{{#resource}}img/some/image.jpeg{{/resource}}">
<img src="{{#sha}}img/image.jpeg{{/sha}}">
</body>
</html>
