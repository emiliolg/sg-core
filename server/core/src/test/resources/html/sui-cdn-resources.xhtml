<div sui-view="tekgenesis.service.html.Views">
    <img src="@Resource(img/image.jpeg)"/>
    <img src="@Sha(img/image.jpeg)"/>
</div>