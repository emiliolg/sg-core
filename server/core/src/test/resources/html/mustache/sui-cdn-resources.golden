<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views">
<head>
    <title>Resources examples</title>
    <link rel="icon" href="/public/sg/img/favicon.png"/>
    <link href="http://cdnd.epi.ninja/css/theme.css">
    <link rel="stylesheet" href="http://cdnd.epi.ninja/sg/concat?/sha/public/css/alpha.css/0125c03ab7&/sha/public/css/beta.css/0a4f910a6e">
    <script src="http://cdnd.epi.ninja/js/lib.js"></script>
</head>
<body>
<img src="http://cdnd.epi.ninja/sha/img/image.jpeg/c4bfa393f1">
<img src="http://cdnd.epi.ninja/img/some/image.jpeg">
<img src="http://cdnd.epi.ninja/img/some/image.jpeg">
<img src="http://cdnd.epi.ninja/public/img/image.jpeg">
</body>
</html>
